src/host/js/utils/enquire.js

```diff
@@ -0,0 +1,29 @@
+import enquire from 'enquire.js';
```

Comment:
What's the size of this lib? Is it lazy loaded at init time, or will this affect page load speed?